### PR TITLE
Remove junit.fdramework usage

### PR DESCRIPTION
**Pull request by bratseth**
Nothing to see here ...

### CODE REVIEW

File: clustercontroller-utils/src/test/java/com/yahoo/vespa/clustercontroller/utils/communication/http/AsyncHttpClientWithBaseTest.java [hunk 1]
```diff
@@ -5,9 +5,7 @@
 import com.yahoo.vespa.clustercontroller.utils.communication.async.AsyncOperationImpl;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotSame;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline clustercontroller-utils/src/test/java/com/yahoo/vespa/clustercontroller/utils/communication/http/AsyncHttpClientWithBaseTest.java]: Wildcard imports are evil..

File: config-model/src/test/java/com/yahoo/vespa/model/builder/xml/dom/chains/search/DomFederationSearcherBuilderTest.java [hunk 2]
```diff
@@ -11,15 +11,13 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import static junit.framework.TestCase.assertEquals;
-import static junit.framework.TestCase.assertNotNull;
-import static junit.framework.TestCase.assertTrue;
 import static org.hamcrest.CoreMatchers.hasItems;
 import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertThat;
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline config-model/src/test/java/com/yahoo/vespa/model/builder/xml/dom/chains/search/DomFederationSearcherBuilderTest.java]: Wildcard import..

File: config-model/src/test/java/com/yahoo/vespa/model/builder/xml/dom/chains/search/DomSearchChainsBuilderTest.java [hunk 3]
```diff
@@ -22,19 +22,18 @@
...
 import static com.yahoo.container.core.ChainsConfig.Chains;
 import static com.yahoo.container.core.ChainsConfig.Components;
-import static junit.framework.TestCase.assertEquals;
-import static junit.framework.TestCase.assertNotNull;
 import static org.hamcrest.CoreMatchers.hasItem;
 import static org.hamcrest.core.Is.is;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.fail;
 import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline config-model/src/test/java/com/yahoo/vespa/model/builder/xml/dom/chains/search/DomSearchChainsBuilderTest.java]: Wildcard import. Please check your settings in IntelliJ

File: config-model/src/test/java/com/yahoo/vespa/model/content/IndexedHierarchicDistributionTest.java [hunk 4]
```diff
@@ -18,15 +18,12 @@
...
-import static junit.framework.TestCase.assertEquals;
-import static junit.framework.TestCase.assertTrue;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertThat;
 import static org.hamcrest.Matchers.containsString;
 import static com.yahoo.vespa.model.content.utils.ContentClusterUtils.createCluster;
 import static com.yahoo.vespa.model.content.utils.ContentClusterUtils.createClusterXml;
 import static com.yahoo.vespa.model.search.utils.DispatchUtils.assertEngine;
 import static com.yahoo.vespa.model.search.utils.DispatchUtils.getDataset;
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline config-model/src/test/java/com/yahoo/vespa/model/content/IndexedHierarchicDistributionTest.java]: Wildcard

File: config-model/src/test/java/com/yahoo/vespa/model/test/PortsMetaTestCase.java [hunk 5]
```diff
@@ -2,14 +2,18 @@
 package com.yahoo.vespa.model.test;
 
 import com.yahoo.vespa.model.PortsMeta;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline config-model/src/test/java/com/yahoo/vespa/model/test/PortsMetaTestCase.java]: Wildcard

File: config/src/test/java/com/yahoo/config/subscription/ConfigSubscriptionTest.java [hunk 6]
```diff
@@ -15,19 +15,14 @@
...
 import java.util.List;
 
-import static junit.framework.TestCase.fail;
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.CoreMatchers.not;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline config/src/test/java/com/yahoo/config/subscription/ConfigSubscriptionTest.java]: Wildcard

File: config/src/test/java/com/yahoo/vespa/config/ConfigDefinitionKeyTest.java [hunk 7]
```diff
@@ -3,11 +3,7 @@
 
 import org.junit.Test;
 
-import static junit.framework.TestCase.assertFalse;
-import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
-
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline config/src/test/java/com/yahoo/vespa/config/ConfigDefinitionKeyTest.java]: Wildcard

File: config/src/test/java/com/yahoo/vespa/config/ConfigDefinitionSetTest.java [hunk 8]
```diff
@@ -3,17 +3,13 @@
 
 import org.junit.Test;
 
-import static junit.framework.TestCase.assertNull;
-import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline config/src/test/java/com/yahoo/vespa/config/ConfigDefinitionSetTest.java]: Wildcard

File: config/src/test/java/com/yahoo/vespa/config/RawConfigTest.java [hunk 9]
```diff
@@ -11,16 +11,13 @@
 import java.util.List;
 import java.util.Optional;
 
-import static junit.framework.TestCase.assertEquals;
-import static junit.framework.TestCase.assertNull;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.not;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline config/src/test/java/com/yahoo/vespa/config/RawConfigTest.java]: Wildcard

File: container-search/src/test/java/com/yahoo/fs4/test/PacketDecoderTestCase.java [hunk 10]
```diff
@@ -11,7 +11,7 @@
 
 import java.nio.ByteBuffer;
 
-import static junit.framework.TestCase.*;
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline container-search/src/test/java/com/yahoo/fs4/test/PacketDecoderTestCase.java]: Wildcard (It was originally as well, but please fix when touching it)

File: container-search/src/test/java/com/yahoo/fs4/test/PacketTestCase.java [hunk 11]
```diff
@@ -6,19 +6,16 @@
 
 import com.yahoo.fs4.*;
 import com.yahoo.search.Query;
-import org.junit.Ignore;
 import org.junit.Test;
 
-import static junit.framework.Assert.assertEquals;
-import static junit.framework.TestCase.*;
-import static org.junit.Assume.assumeTrue;
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline container-search/src/test/java/com/yahoo/fs4/test/PacketTestCase.java]: Wildcard

File: container-search/src/test/java/com/yahoo/prelude/cache/test/CacheTestCase.java [hunk 12]
```diff
@@ -1,24 +1,26 @@
...
-
 import com.yahoo.search.result.Hit;
 import com.yahoo.search.Query;
 import com.yahoo.search.Result;
 import com.yahoo.statistics.Statistics;
 import com.yahoo.prelude.cache.Cache;
 import com.yahoo.prelude.cache.QueryCacheKey;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (gjoranv) [inline container-search/src/test/java/com/yahoo/prelude/cache/test/CacheTestCase.java]: Wildcard

**Review by gjoranv — CHANGES_REQUESTED** [review]
Please check your IDEA settings:
Editor -> Code style -> Java -> Imports (tab)

I suggest to set an insanely high number for 'Class count' and 'Names count'

Tip (you probably already know) to optimize imports: Ctrl+Alt+O

**Comment by bratseth** [comment]
New Mac problems. Fixed & merging.

**Comment by gjoranv** [comment]
👍

**Pull request merged**